specification/quota/resource-manager/Microsoft.Quota/stable/2021-03-15/examples/putMachineLearningServicesQuotaRequestDedicated.json

```diff
@@ -0,0 +1,55 @@
+{
+  "parameters": {
+    "scope": "subscriptions/D7EC67B3-7657-4966-BFFC-41EFD36BAAB3/providers/Microsoft.MachineLearningServices/locations/eastus",
+    "resourceName": "StandardDv2Family",
+    "api-version": "2021-03-15",
+    "createQuotaRequest": {
+      "properties": {
+        "limit": 200,
+        "unit": "Count",
+        "name": {
+          "value": "StandardDv2Family"
+        },
+        "resourceType": "dedicated"
+      }
+    }
+  },
+  "responses": {
+    "201": {
+      "body": {
+        "id": "/subscriptions/D7EC67B3-7657-4966-BFFC-41EFD36BAAB3/providers/Microsoft.Quota/resourceProviders/Microsoft.MachineLearningServices/locations/eastus/serviceLimitsRequests/2B5C8515-37D8-4B6A-879B-CD641A2CF605",
+        "name": "2B5C8515-37D8-4B6A-879B-CD641A2CF605",
+        "type": "Microsoft.Quota/ServiceLimits",
+        "properties": {
+          "provisioningState": "InProgress",
+          "message": ""
+        }
+      }
+    },
+    "200": {
+      "body": {
+        "id": "/subscriptions/D7EC67B3-7657-4966-BFFC-41EFD36BAAB3/providers/Microsoft.Quota/resourceProviders/Microsoft.MachineLearningServices/locations/eastus/serviceLimitsRequests/2B5C8515-37D8-4B6A-879B-CD641A2CF605",
+        "name": "2B5C8515-37D8-4B6A-879B-CD641A2CF605",
+        "type": "Microsoft.Quota/serviceLimits",
+        "properties": {
+          "provisioningState": "Succeeded",
+          "message": "Request succeeded",
+          "properties": {
+            "properties": {
+              "limit": 200,
+              "unit": "Count",
+              "currentValue": 160,
+              "name": {
+                "value": "StandardDv2Family",
```

Comment:
They are specified in the outer "properties" property in the request body. The response should return them as in the original request (returning additional properties is okay).

See the rpc: [https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/resource-api-reference.md#put-resource](https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/resource-api-reference.md#put-resource)

"The response body should contain at least the original request that was PUT (and any other properties that would be returned in a GET, such as provisioningState, name, Id and type)."

Reply:
The 200 response contains the original request:
"200": {
      "body": {
        "id": "/subscriptions/D7EC67B3-7657-4966-BFFC-41EFD36BAAB3/providers/Microsoft.Compute/locations/eastus/providers/Microsoft.Quota/quotaLimitsRequests/2B5C8515-37D8-4B6A-879B-CD641A2CF605",
        "name": "standardFSv2Family",
        "type": "Microsoft.Quota/serviceLimits",
        "properties": {
          "provisioningState": "Succeeded",
          "message": "",
          "properties": {
            "properties": {
              _"limit": 200,
              "unit": "Count",
              "currentValue": 160,
              "name": {
                "value": "standardFSv2Family",
                "localizedValue": "Standard FSv2 Family vCPUs"
              },_
              "quotaPeriod": "",
              "properties": {}
            }
          }
        }
      }

**Is it regarding 202 response?:**
"202": {
      "body": {
        "id": "/subscriptions/D7EC67B3-7657-4966-BFFC-41EFD36BAAB3/providers/Microsoft.Compute/locations/eastus/providers/Microsoft.Quota/quotaLimitsRequests/2B5C8515-37D8-4B6A-879B-CD641A2CF605",
        "name": "standardFSv2Family",
        "type": "Microsoft.Quota/ServiceLimits",
        "properties": {
          "provisioningState": "InProgress",
          "message": ""
        }
      }
    },

Reply:
Updated the schema as suggested.

Reply:
Updated the schema to match the example suggested:

_{
"id": "/subscriptions/D7EC67B3-7657-4966-BFFC-41EFD36BAAB3/providers/Microsoft.Quota/resourceProviders/Microsoft.MachineLearningServices/locations/eastus/quotaLimitsRequests/2B5C8515-37D8-4B6A-879B-CD641A2CF605",
"name": "StandardDv2Family",
"type": "Microsoft.Quota/serviceLimits",
"properties": {
"provisioningState": "Succeeded",
"message": "Request succeeded",
"limit": 200,
"unit": "Count",
"currentValue": 160,
"name": {
"value": "StandardDv2Family",
"localizedValue": "Standard Dv2 Family Cluster Dedicated vCPUs"
},
"resourceType": "dedicated",
"quotaPeriod": "",
"properties": {}
}
}_